upgrade_guide/migrating_from_astyanax/queries_and_results/README.md

```diff
@@ -1,3 +1,7 @@
+```eval_rst
+:orphan:
+```
```

Comment:
Are this addition a must?
It will make sync with upstream hard
If no other way, lets disable the warning it generate

Reply:
It's not a must, but Sphinx throws the warning ``WARNING: document isn't included in any toctree``. This does not stop the documentation from the building, so I have opted to delete the orphan references.